parm/data_locations.yml

```diff
@@ -386,6 +385,6 @@ NOHRSC_obs:
       - "dcom_{yyyy}{mm}{dd}.tar"
     file_names:
       obs:
-        - "sfav2_CONUS_*h_{yyyy}{mm}{dd}{hh}_grid184.grb2"
+        - "sfav2_CONUS_6h_{yyyy}{mm}{dd}*_grid184.grb2"
```

Comment:
Do we need to update the staged data to use `sfav2_CONUS_6h_{yyyy}{mm}{dd}*_grid184.grb2` rather than `sfav2_CONUS_06h_{yyyy}{mm}{dd}*_grid184.grb2`?  I see in the `MET_verification_winter_wx` configuration file that you made changes to use the current `06h` structure.  Should we just go ahead and update the staged data so that it is universally using `6h` (no leading `0`)?

Reply:
@MichaelLueken Let me give some background first for the changes on this line:
1) The change from `*h` to `6h` is so that only the NOHRSC obs file for 6-hour accumulation of ASNOW is ever retrieved.  That's because this PR changes the workflow so that 24-hour accumulated ASNOW obs are now obtained by adding up the 6-hour ones instead of directly retrieving the 24-hour NOHRSC ASNOW obs (so now we do for NOHRSC ASNOW what we've been doing all along for CCPA APCP obs).  We decided to do it this way in one of our DTC meetings because it is closer to what the model outputs; the models usually do not account for compaction of snow over time, and adding up the 6-hour ASNOW accumulation obs gives a result closer to the models' output than retrieving 24-hour ASNOW obs, which naturally incorporates more compaction of snow.
2) The change from `{hh}` to `*` is so that when calling the `retrieve_data.py` script in the new `get_obs.py` script, all four `sfav2_...` files in a given NOHRSC archive (tar) file are extracted in one go, i.e. in the same call to `retrieve_data.py`.  (The four obs files are at hours-of-day 00, 06, 12, and 18.).  Currently, each of these obs files is retrieved in a separate call to `retrieve_data.py`, and that is much slower because usually the majority of the wallclock time is spent making a connection to HPSS (whereas the 4 obs files themselves are tiny, on the order of a couple of hundred kb's and so are very quick to retrieve).

The setting of `*h` vs. `6h` here doesn't affect the file that the `get_obs` tasks look for on disk.  That is determined by the config variable `OBS_NOHRSC_ASNOW_FN_TEMPLATE`.  If you look in the config file for the test `MET_ensemble_verification_winter_wx` (which is I think what you meant, not `MET_verification_winter_wx`), you'll see the setting:
```
  OBS_NOHRSC_ASNOW_FN_TEMPLATE: '{%- set data_intvl_hrs = "%02d" % NOHRSC_OBS_AVAIL_INTVL_HRS %}
                                 {{- "{valid?fmt=%Y%m%d}/sfav2_CONUS_" ~ data_intvl_hrs ~ "h_{valid?fmt=%Y%m%d%H}_grid184.grb2" }}'
```
You can see from the `%02d` that this is looking for a file with `06h` in it.  Since that's what we have on Hera now, it will find it.  If you remove the symlink with just `6h` in it, this will still work.  I'd actually rather leave the `06h` file on disk since it provides a good test of whether setting this `OBS_NOHRSC_ASNOW_FN_TEMPLATE` variable actually works.  [Note that the **default** behavior is to use as few digits as possible for the accumulation in the file name; see the definition of `OBS_NOHRSC_ASNOW_FN_TEMPLATE` variable in `config_defaults.yaml`, where the formatting `"%d"` is used instead.  So the test `MET_ensemble_verification_winter_wx` is checking whether doing something other than the default actually works.]

So, to conclude:  I'd prefer if we leave the `_06h` in the file name and just remove the `_6h`symlink that I had you create earlier.  Thanks!

Reply:
Thank you very much for your explanation, @gsketefian!  I have gone ahead and removed the `_6h` symlink on Hera and it makes sense to keep the current modification in the `MET_ensemble_verification_winter_wx` configuration to ensure that `OBS_NOHRSC_ASNOW_FN_TEMPLATE` continues to behave correctly.